File: hubz-frontend/src/types/tag.ts
export interface Tag {
  id: string;
  name: string;
  color: string;
  organizationId: string;
  createdAt: string;
}

export interface CreateTagRequest {
  name: string;
  color: string;
}

export interface UpdateTagRequest {
  name?: string;
  color?: string;
}

File: hubz-backend/src/main/java/com/hubz/domain/model/Tag.java
package com.hubz.domain.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Tag {

    private UUID id;
    private String name;
    private String color;
    private UUID organizationId;
    private LocalDateTime createdAt;
}
